src/main/resources/skillresponse/response.properties

```diff
@@ -116,13 +119,16 @@ askForCityAfterStop.personalized={0}, You haven''t set your region yet. In what
 duplicatedStops.personalized=We found {0} stops associated with the stop number. Did you mean the {1} stop?
 #{0} stopName
 duplicatedStopConfirm.personalized=Ok, what about the {0} stop?
+#{1} cityName, {2} stopNumber
+copyProfile.personalized=Hi {0}, We found an existing profile living in {1} and the stop number is {2}, do you want to use that?
```

Comment:
`{2}` currently uses the persisted `stopId` from DynamoDB - this needs to change, as that ID is not user-facing and in some cases can be nonsense like a UUID or even worse a different integer value than the user-facing ID. In other words, the user could successfully set their stop to the ID `1234`, but we'd read back to them `5678`. The field you want to use for user-facing ID is "stop code" (e.g., from the `ObaStopResponse.getCode()`). We retain this in the session when onboarding the user, and if the user asks for their current stop we have to fetch it from the server (see `GetStopNumberIntentHandler`.

Same for any other user-facing presentation of stop ID - I haven't had a chance to review all the other uses.

Reply:
The only place using the stopId from the DynamoDB is on Launch, I was wondering any pattern for this ID so that I can replace? or just saying the stop name if there is no easy way?

Reply:
There isn't any pattern for converting stop IDs to stop codes.

I'd suggest hitting the server to get the stop code and stop name from the stop ID, which is what we do for "Ask OneBusAway what is my stop?".

~~~
final ObaStopResponse stop = obaUserClient.getStopDetails(obaUserDataItem.getStopId());
String stopCode = stop.getStopCode();
~~~

And then I'd suggest treating the response the same way we treat the "Ask OneBusAway what is my stop?" response, which would be:

`Hi {0}, OneBusAway now supports personalized profiles. Your current settings have you living in {1} and the stop is {2}, {3}. Do you want to use that?`

...where `{2}` is the stop code and `{3}` is the stop name.

Reply:
If there is no easy way, I will just make another to OBA to get it, thanks